Sources/Vortex/System/VortexSystem.swift

```diff
@@ -8,7 +8,7 @@
 import SwiftUI
 
 /// The main particle system generator class that powers Vortex.
-public class VortexSystem: Codable, Equatable, Hashable {
+public class VortexSystem: Codable, Identifiable, Equatable, Hashable {
```

Comment:
Adding `Identifiable` here (and the `public var id` change below) is a great change – thank you!